### PR TITLE
Add fritzing

### PR DESCRIPTION
**Pull request by gitolicious**
A popular tool for hardware tinkerers to layout their sketches and PCBs

### CODE REVIEW

File: fritzing.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    },
+    "bin": "Fritzing.exe",
+    "shortcuts": [
+      [
+        "Fritzing.exe",
+        "Fritzing"
+      ]
+    ],
+    "checkver": {
+        "url": "https://github.com/fritzing/fritzing-app/releases/latest",
```
[the remark this answers is not in the document]
Comment (gitolicious) [reply fritzing.json]: Had some troubles with this repo when using the github link directly as their quirky version naming scheme (0.9.3b) is not supported by the current default implementation.

https://github.com/lukesampson/scoop/blob/1349419c2f5abe7e7b7845ef724095029adc11e8/bin/checkver.ps1#L108

[the remark this answers is not in the document]
Comment (Ash258) [reply fritzing.json]: That's why you keep that regex.

**Comment by gitolicious** [comment]
Stupid me deleted my fork without knowing my PR will end up orphaned... Sorry about that. Will close this one and open a new PR.

**Comment by r15ch13** [comment]
Please change `license` to `GPL-2.0-or-later,GPL-3.0-or-later,CC-BY-SA-3.0`

**Comment by r15ch13** [comment]
Replaced by https://github.com/lukesampson/scoop-extras/pull/1501

**Pull request closed without merging**